src/ui/layout/index.md

```diff
@@ -1250,9 +1250,9 @@ The following resources might help when writing layout code.
 [`ListTile`]: {{api}}/material/ListTile-class.html
 [`ListView`]: {{api}}/widgets/ListView-class.html
 [`MainAxisAlignment`]: {{api}}/rendering/MainAxisAlignment.html
-[Material card]: {{site.material}}/design/components/cards.html
-[Material Design]: {{site.material}}/design
-[Material Design palette]: {{site.material}}/design/color
+[Material card]: {{site.material}}/components/cards
+[Material Design]: {{site.material}}/styles
+[Material Design palette]: {{site.material}}/styles/color
```

Comment:
I don't think we offer a palette anymore, consider rewriting

Reply:
Can I talk about the dynamic colors over there?
https://pub.dev/packages/dynamic_color

Reply:
@guidezpl

Reply:
I would remove this example entirely